dagshub/common/cli.py

```diff
@@ -105,8 +114,77 @@ def upload(ctx,
 
     owner, repo_name = repo
     repo = Repo(owner=owner, name=repo_name, branch=branch)
-    repo.upload(file=filename, path=target, commit_message=message, force=update)
+    try:
+        repo.upload(file=filename, path=target, commit_message=message, force=update)
+    except RuntimeError as re:
+        # if uploaded filename already exists but update is False, a strange error message will appear
+        # since gogs tries to check last commit in this case
+        if "invalid last_commit" in str(re):
+            raise RuntimeError('Verify that the uploaded file is new') from re
+        else:
+            raise re
+
+
+@cli.command()
+@click.argument("repo", callback=validate_repo)
+@click.option("-u", "--upload-data", help="Upload data from specified url to new repository")
+@click.option("-c", "--clone", is_flag=True,  help="Clone repository locally")
+@click.option("-v", "--verbose", default=0, count=True, help="Verbosity level")
+def repo_create(
+                repo,
+                upload_data,
+                clone,
+                verbose):
+    """
+    repo_create
+    example:  dagshub repo-create sdafni.yd/yuvtutorial14 -u "http://0.0.0.0:8080/index.html" --checkout
+    """
+    logger = logging.getLogger()
+    logger.setLevel(to_log_level(verbose))
+
+    owner, repo_name = repo
+    INITIAL_DATA_FILE_NAME = 'repo_created_initial_data'
```

Comment:
No need to ask the user for their username if they can only create the repo for themselves or an organization they're in.
You can actually get the full name of the repo if you use [`repo.full_name`](https://github.com/DagsHub/client/blob/master/dagshub/upload/wrapper.py#L332)

There is the edge case of organizations, but that would require another flag

Reply:
done (pending push)